src/javascript/app_2/Stores/Modules/Contract/contract_store.js

```diff
@@ -80,7 +87,8 @@ export default class ContractStore extends BaseStore {
     @action.bound
     updateProposal(response) {
         if ('error' in response) {
-            this.has_error = true;
+            this.has_error     = true;
+            this.error_message = response.error.message;
```

Comment:
There are cases that `proposal_open_contract` does not return an error but an empty object in response: `proposal_open_contract: {}`.

This happens when trying to view a `contract_id` which doesn't exist or doesn't belong to the current client.
We need to handle those cases as well.

Thanks.